Lib/fontTools/varLib/designspace.py

```diff
@@ -1,5 +1,6 @@
 """Rudimentary support for loading MutatorMath .designspace files."""
 from __future__ import print_function, division, absolute_import
+import collections
```

Comment:
I guess if you use `OrderedDict` more than once, you can spare an extra attribute lookup if you import `from`, as Miguel suggested.